editor/plugins/script_editor_plugin.cpp

```diff
@@ -3174,7 +3175,6 @@ ScriptEditor::ScriptEditor(EditorNode *p_editor) {
 	add_child(find_in_files_dialog);
 	find_in_files = memnew(FindInFilesPanel);
 	find_in_files_button = editor->add_bottom_panel_item(TTR("Search Results"), find_in_files);
-	find_in_files_button->set_tooltip(TTR("Search in files"));
```

Comment:
Why remove the tooltip? IMO all buttons should have tooltips.

Reply:
It's literally the only bottom panel button that has one.

Reply:
OK, fine with removing it for now then as it doesn't convey a lot of information anyway. But tooltips should likely be added eventually to all bottom panel buttons.